### PR TITLE
[MODULAR] Refactors MCR designs

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

as above
less copypasta
clean code is good code
Cells are now in the Ammo category (basic is in Security in autolathe)
All other parts are in Weapons category in seclathe.
Removed designs from scilathe and mechfab, Gandalf said it was "an oversight"

## How This Contributes To The Skyrat Roleplay Experience

MCR parts in more consistent places in the lathes with other gun stuff, less bloat where it shoudn't be seen.

## Changelog

:cl:
qol: Moved MCR designs around. Cells are now Ammo, Gun Parts are now Weapons category.
refactor: MCR lathe design stuff
fix: A bug in one of the material costs for one of the camos
/:cl:

### CODE REVIEW

**Comment by MassacreDemigirl** [comment]
this will be a pain for #11377

**Comment by YakumoChen** [comment]
oops, I didn't realize you had an MCR thing that big. 

Well, all the better then, since I'd have had to go back to yours anyway.

**Comment by MassacreDemigirl** [comment]
I already made my pr compliant with this refactor so, keep this one up

File: modular_skyrat/modules/microfusion/code/microfusion_designs.dm [hunk 1]
```diff
@@ -1,228 +1,173 @@
...
+	name = "Microfusion Part"
+	build_type = PROTOLATHE | AWAY_LATHE
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+	construction_time = 10 SECONDS //dunno if this is for mechfabs or what but I'll keep this anyway
+	category = list("Weapons")
+
+// EMITTERS
+/datum/design/microfusion/enhanced_phase_emitter
+	name = "Enhanced Microfusion Phase Emitter"
+	desc = "The core of a microfusion projection weapon, produces the laser."
```
Comment (jjpark-kb) [inline modular_skyrat/modules/microfusion/code/microfusion_designs.dm]: I recommend creating another parent called
> /datum/design/microfusion/emitter

Where it has the desc and the materials. You'll notice you have three designs that have the same desc and materials, which could be condensed with this new parent.

Comment (jjpark-kb) [reply modular_skyrat/modules/microfusion/code/microfusion_designs.dm]: What this means is that the new path for this enhanced phase emitter will be 
> /datum/design/microfusion/emitter/enhanced_phase_emitter

or something along those lines.

Comment (YakumoChen) [reply modular_skyrat/modules/microfusion/code/microfusion_designs.dm]: I was thinking about splitting up each set of parts more, but it really isn't needed, the vars in the /microfusion/ parent apply to everything else I wanted included, the only thing I'd be knocking down is the desc var.

The materials are also not the same between them. I could give them better descriptions but I was lazy and the names already kind of speak for themselves. Design descriptions are rarely actually seen regardless.

**Comment by YakumoChen** [comment]
redone since #11377 was merged before this. Added more sorting.

**Pull request merged**